safe_transaction_service/contracts/tx_decoder.py

```diff
@@ -590,6 +614,12 @@ def get_abi_function(
                     # Otherwise we fall back to the general abi that matches the selector
                     return contract_selectors_with_abis[selector]
             return self.fn_selectors_with_abis[selector]
+        # Check if the contract has a fallback call and return a minimal ABIFunction for fallback call
+        elif address and self.contract_has_fallback_call(address):
+            return ABIFunction(
```

Comment:
You need to get the proper fallback function. Fallback can also take `bytes` as `input` or `output`: https://solidity-by-example.org/fallback/

Reply:
Understood, Updated [here](https://github.com/safe-global/safe-transaction-service/pull/2262/commits/4caf91e5a242ebb07a6d3ff2cf11d225dbefc8da).